python/cudf/cudf/core/index.py

```diff
@@ -2544,6 +2544,47 @@ def tz_localize(self, tz, ambiguous="NaT", nonexistent="NaT"):
             result_col = localize(self._column, tz, ambiguous, nonexistent)
         return DatetimeIndex._from_data({self.name: result_col})
 
+    def tz_convert(self, tz):
+        """
+        Convert tz-aware datetimes from one time zone to another.
+
+        Parameters
+        ----------
+        tz: str
+            Time zone for time. Corresponding timestamps would be converted
```

Comment:
The pandas docstring is worded in a weird way. However, I would leave this as-is to match pandas.